types/backbone/index.d.ts

```diff
@@ -101,20 +107,19 @@ declare namespace Backbone {
 
     class ModelBase extends Events {
         url: any;
-        parse(response: any, options?: any): any;
         toJSON(options?: any): any;
         sync(...arg: any[]): JQueryXHR;
     }
 
-    class Model extends ModelBase {
+    class Model<TAttributes extends {}> extends ModelBase {
```

Comment:
I don't think you want `extends {}` here, as everything extends `{}` (even `string` or `number`) so you may as well omit it. Perhaps `object` is what you want if you want to exclude primitive types.

Reply:
Right.